Prog/mt_vern_profile_data.sas

```diff
@@ -341,11 +345,11 @@ data sales_res;
 	saleyear=year(saledate);
 run;
```

Comment:
@mcohenui should adjust for inflation first.

 %dollar_convert( originalvar, newvar, startyear, 2017, series=CUUR0000SA0L2 )

Reply:
also only use from 2002 forward....data before that are less accurate.

Reply:
@mcohenui - you needed to put in saleyear not 2003 (otherwise you were treating all sales as if they were in 2003 dollars.)